Muhet/MYBRAND
# { features } Ft- create contact js 124

#### What does this PR do?
[This PR is for creating the function of contact form. sanding the message to the admin from client side.]
#### Description of Task to be completed
[to be able to write the message and send to admin]
#### How should this be manually tested?
[Fill the contact form and hit save button and see if you will get notification which says "Your message received"]
#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
[Ft- create contact js 124]
#### Screenshots (if appropriate)
![contactForm](https://user-images.githubusercontent.com/86248137/221683423-c46cd293-101a-48e5-82a6-944b8be5451f.JPG)
